pkg/server/plugin/upstreamauthority/awspca/pca.go

```diff
@@ -216,6 +226,9 @@ func (m *PCAPlugin) MintX509CA(request *upstreamauthority.MintX509CARequest, str
 
 	// The last certificate returned from the chain is the bundle.
 	bundle := [][]byte{certChain[len(certChain)-1].Raw}
+	for _, upstream := range m.trustBundle {
+		bundle = append(bundle, upstream.Raw)
+	}
```

Comment:
Might want to dedupe here, since at least in the use-cases I'm thinking, the upstream root is likely also in the user-supplied bundle.

Reply:
Or, only use the on-disk bundle, but emit an Error level log if the root from PCA isn't in the bundle

Reply:
Maybe a combination of the two? I don't think we could just rely on the on-disk bundle since that would impact back-compat for the existing configurations which don't specify the `bundle_file_path`.